MAUI/DataGrid/selection.md

```diff
@@ -57,6 +99,26 @@ The `SfDataGrid` provides [SelectedIndex](https://help.syncfusion.com/cr/maui/Sy
 
 `SfDataGrid.SelectedRows`: Provides all the selected records of the selected items when multiple selection is enabled.
 
+### Getting selected cells.
+
+The `SfDataGrid` provides[`SfDataGrid.GetSelectedCells()`] and [`SfDataGrid.GetSelectedCellInfo()`].
+
+`SfDataGrid.GetSelectedCells()` : Provides all the selected cell, when multiple cells are selected.
```

Comment:
Provides details of all selected cells when multiple cells are selected

Reply:
Addressed the changes